lib/pbkit/pbkit.c

```diff
@@ -2549,14 +2492,9 @@ void pb_fill(int x, int y, int w, int h, DWORD color)
     //color=((color>>8)&0xF800)|((color>>5)&0x07E0)|((color>>3)&0x001F);
 
     p=pb_begin();
-    pb_push(p++,NV20_TCL_PRIMITIVE_3D_CLEAR_VALUE_HORIZ,2);     //sets rectangle coordinates
-    *(p++)=((x2-1)<<16)|x1;
-    *(p++)=((y2-1)<<16)|y1;
-    pb_push(p++,NV20_TCL_PRIMITIVE_3D_CLEAR_VALUE_DEPTH,3);     //sets data used to fill in rectangle
-    *(p++)=0;           //(depth<<8)|stencil
-    *(p++)=color;           //color
-    *(p++)=0xF0;            //triggers the HW rectangle fill (0x03 for D&S)
-    pb_end(p);
+    p=pb_push2(p,NV20_TCL_PRIMITIVE_3D_CLEAR_VALUE_HORIZ,((x2-1)<<16)|x1,((y2-1)<<16)|y1); //sets rectangle coordinates
+    p=pb_push3(p++,NV20_TCL_PRIMITIVE_3D_CLEAR_VALUE_DEPTH,0,color,0xF0); //sets data used to fill in rectangle
```

Comment:
This is a case where 3 different commands are being submitted. \- so avoid this step.

*(See my other note about merging `pb_push` into a single line)*

Reply:
Reverted. :+1: